doc/containers.md

```diff
@@ -595,7 +599,7 @@ empty (default), no snapshots will be created. `snapshots.schedule.stopped`
 controls whether or not stopped container are to be automatically snapshotted.
 It defaults to `false`. `snapshots.pattern` takes a pongo2 template string,
 and the pongo2 context contains the `creation_date` variable. Be aware that you
-should format the date (e.g. use `{{ creation_date|date:"2006-01-02_15-04-05" }}`) 
+should format the date (e.g. use `{{ creation_date|date:"2006-01-02_15-04-05" }}`)
```

Comment:
Seems unrelated to this change and a trailing whitespace in markdown indicates a line break, so this may actually have made things worse :)

Reply:
@stgraber ah, looks like I included this accidentally after vscode auto removed the line, will take out